cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -271,11 +294,35 @@ def is_displayed(self):
         )
 
 
-class BaseCatalogItem(BaseEntity, Updateable, Pretty, Taggable):
+class ButtonDetailView(ServicesCatalogView):
+    title = Text("#explorer_title_text")
+
+    button_type = SummaryFormItem("Basic Information", "Button Type")
+    playbook_cat_item = SummaryFormItem("Basic Information", "Ansible Playbook")
+    target = SummaryFormItem("Basic Information", "Target")
+    text = SummaryFormItem(
+        "Basic Information",
+        "Text",
+        text_filter=lambda text: re.sub(r"\s+Display on Button\s*$", "", text),
+    )
+    hover = SummaryFormItem("Basic Information", "Hover Text")
+    dialog = SummaryFormItem("Basic Information", "Dialog")
+    system = SummaryFormItem("Object Details", "System/Process/")
+    message = SummaryFormItem("Object Details", "Message")
+    request = SummaryFormItem("Object Details", "Request")
+    type = SummaryFormItem("Object Attribute", "Type")
+    show = SummaryFormItem("Visibility", "Show")
 
     @property
-    def button_icon_name(self):
-        return 'broom'
+    def is_displayed(self):
+        return (
+            self.in_explorer and
+            self.catalog_items.is_opened and
+            "Button" in self.title.text
```

Comment:
Question: Is there anything specific about the `Button` that can be checked against?

Reply:
yup, but this view not align with any entity or object so we can not use `context` objects.